packages/nouns-subgraph/src/nouns-dao.ts

```diff
@@ -154,9 +153,18 @@ export function handleVoteCast(event: VoteCast): void {
 
   if (event.params.support == 0) {
     proposal.againstVotes = proposal.againstVotes.plus(event.params.votes);
+  } else if (event.params.support == 1) {
+    proposal.forVotes = proposal.forVotes.plus(event.params.votes);
+  } else if (event.params.support == 2) {
+    proposal.abstainVotes = proposal.abstainVotes.plus(event.params.votes);
   }
 
-  if (proposal.usingDynamicQuorum) {
+  const dqParams = getOrCreateDynamicQuorumParams();
+  const usingDynamicQuorum =
+    dqParams.dynamicQuorumStartBlock !== null &&
```

Comment:
I don't believe that it matters when comparing against `null`, but it's best practice to use `==` and `!=` in AssemblyScript comparisons. `===` and `!==` compare references in AssemblyScript.

Reply:
There are 3 instances of this in the subgraph that should be updated just in case.